scripts/installer

```diff
@@ -482,6 +482,7 @@ help () {
   echo -e "\t[--openshift]\t\t\t\tWill build manifests for openshift"
   echo -e "\t[--image-stream]\t\t\tWill generate manifests using openshift image stream"
   echo -e "\t[--read-only]\t\t\t\tWill build manifests for a readonly deployment"
+  echo -e "\t[--logout-url]\t\t\t\tWill set up the logout URL"
```

Comment:
Should this be 
```suggestion
  echo -e "\t[--logout-url <logout-url>]\t\t\t\tWill set up the logout URL"
```
since a URL is needed?

Reply:
👍
fixed, i updated docs too